test/util/certs/cert.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
don't commit certs

the whole point of this is to not have certs committed to the repo at all